kaldb/src/main/java/com/slack/kaldb/logstore/LogMessage.java

```diff
@@ -117,9 +115,7 @@ && getType() != null
 
   private BadMessageFormatException raiseException(Throwable t) {
     throw new BadMessageFormatException(
-        String.format(
-            "Index:%s, Type: %s, Id: %s, Source: %s".format(getIndex(), getType(), id, source)),
-        t);
+        "Index:%s, Type: %s, Id: %s, Source: %s".format(getIndex(), getType(), id, source), t);
```

Comment:
This looks like a refactor went wrong, as this is invalid syntax (static method accessed via an instance reference). IntelliJ should also have highlighted this as incorrect.

```java
System.out.println("foo %s".format("bar"));
```
will output `bar`, not `foo bar`.

Reply:
Yeah, looks like the refactor is wrong. Fixed.